charmhelpers/contrib/openstack/context.py

```diff
@@ -2562,6 +2566,18 @@ def _numa_node_cores(self):
                 nodes[index] = self._parse_cpu_list(cpulist.read().strip())
         return nodes
 
+    def _lowest_cpu_mask(self, num_cores=1):
```

Comment:
Extracted computation of lowest mask in order to be able to re-use it for the default mask generation of pmd-cpu-mask.